packages/truffle/test/scenarios/commandRunner.js

```diff
@@ -42,5 +43,36 @@ module.exports = {
         reject(child.error);
       }
     });
-  }
+  },
+  runInDevelopEnvironment: (commands = [], config) => {
+    let child;
+    // open `truffle develop`
+    const cmdLine = "node " +
+      path.join(__dirname, "../../../", "core", "cli.js") +
+      " develop";
+
+    return new Promise((resolve, reject) => {
+      child = exec(cmdLine, { cwd: config.working_directory });
+
+      if (child.error) return reject(child.error);
+
+      child.stderr.on("data", (data) => {
+        config.logger.log("ERR: ", data);
```

Comment:
This is a good idea to include these prefixes. They'll be useful when debugging!